src/cluster_legacy.c

```diff
@@ -2870,6 +2870,9 @@ int clusterProcessPacket(clusterLink *link) {
     uint16_t flags = ntohs(hdr->flags);
     uint64_t senderCurrentEpoch = 0, senderConfigEpoch = 0;
     clusterNode *sender = getNodeFromLinkAndMsg(link, hdr);
+    int sender_claims_primary = !memcmp(hdr->replicaof, CLUSTER_NODE_NULL_NAME, CLUSTER_NAMELEN);
+    int sender_was_replica = sender && nodeIsReplica(sender);
+    int sender_was_primary = sender && nodeIsPrimary(sender);
```

Comment:
I thought `sender_claims_primary` is `sender_claims_to_be_primary` but sure I can go with the longer name too, especially when cluster states are involved.

Reply:
Yes, I think `sender_claims_to_be_primary` is easier and faster to understand.

Most often, the word claim is used when you claim some object, like a slot. To claim that you are something is not the most common use of the word claim. With "claims to be", it's immediately very clear to me. I'd understand the variable name in 1 seconds instead of 5.

Reply:
sounds good. will update to `sender_claims_to_be_primary`